src/include/OSL/dual.h

```diff
@@ -61,7 +61,7 @@ class Dual {
 
     /// Default ctr leaves everything uninitialized
     ///
-    constexpr Dual () { }
+    OIIO_CONSTEXPR14 Dual () { }
```

Comment:
THAT'S a weird one! 

Do you have any idea why MSVC would think that an empty ctr isn't C++11-constexpr compliant?

Reply:
known issue in msvc2015, the release notes *1 said they wanted to fix it in a future update, but it doesn't look like they ever did (i build with 2015 update 5) haven't tested with msvc2017 yet. 

*1 https://blogs.msdn.microsoft.com/vcblog/2015/04/29/c11-constant-expressions-in-visual-studio-2015-rc/